src/lib/canvas/dropzone.js

```diff
@@ -45,9 +45,8 @@ class Dropzone extends React.Component {
 
   componentDidMount() {
     const container = ReactDOM.findDOMNode(this);
-    this.setState({
-      ratio: isSmallSize(container) ? 1 : SECTORS_CENTER_RATIO
-    });
+    
+    this.setState({ ratio: isSmall(container) ? 1 : SECTORS_CENTER_RATIO });
```

Comment:
If I use it in the constructor I will get an error:
`Unable to find node on an unmounted component.`
because I'm using `ReactDOM.findDOMNode(this)` to find the container.